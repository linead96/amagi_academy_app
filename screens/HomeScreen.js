import React from 'react';
import {
  StyleSheet,
  View,
  Text,
} from 'react-native';

const styles = StyleSheet.create({
  container: {
    flex: 1,
    fontSize: 20,
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: '#fff',
  },
});

export default class HomeScreen extends React.Component {
  static navigationOptions = {
    title: 'Home',
  };

  render() {
    return (
      <View style={styles.container}>
        <Text>Amagi Academy</Text>
      </View>
    );
  }
}
