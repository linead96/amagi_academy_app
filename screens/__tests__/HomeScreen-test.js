import 'react-native';
import React from 'react';
import HomeScreen from '../HomeScreen';
import renderer from 'react-test-renderer';

test('renders flash card initial screen', () => {
    const tree =renderer.create(<HomeScreen/>).toJSON();
    expect(tree).toMatchSnapshot();
});