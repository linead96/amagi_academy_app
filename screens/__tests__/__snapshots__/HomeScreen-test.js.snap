// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders flash card initial screen 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "fontSize": 20,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Amagi Academy
  </Text>
</View>
`;
